temporal-sdk/src/main/java/io/temporal/common/interceptors/WorkflowOutboundCallsInterceptorBase.java

```diff
@@ -167,6 +168,11 @@ public void upsertMemo(Map<String, Object> memo) {
     next.upsertMemo(memo);
   }
 
+  @Override
+  public Scope getMetricsScope() {
+    return next.getMetricsScope();
```

Comment:
Took a sec for me to realize `SyncWorkflowContext` already has `getMetricsScope`. May want to toss an `@Override` on it, but no biggie.